src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/DataArchiveValidationServiceTests.cs

```diff
@@ -0,0 +1,7 @@
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
+{
+    public class DataArchiveValidationServiceTests
+    {
+        
```

Comment:
Seems like we're missing tests for this?

Reply:
done